### DISCUSSION TITLE
group-invalid not working

### DISCUSSION THREAD

**Discussion by hornta** [tailwindlabs/tailwindcss#12790] [Help] [answered]
https://play.tailwindcss.com/fk8IgqQqWE

I expect this text to be red but it's green. I tried to extend config with invalid but it didn't change anything.

```html
<div aria-invalid="true" class="group">
  <div class="group-valid:text-green-800 group-invalid:text-red-800">text</div>
</div>

```

**Comment by wongjn** [comment, marked as answer]
`group-invalid` maps to the [`:invalid`](https://tailwindcss.com/docs/hover-focus-and-other-states#invalid) CSS psuedo-class, not `aria-invalid="true"`. For `aria-invalid="true"`, use `group-aria-[invalid=true]` as per [this Tailwind Play](https://play.tailwindcss.com/H0nw3LhS3P).

**Reply by hornta** [reply]
Ah thank you, I totally forgot CSS had that, haha.
